woocommerce/woocommerce-gutenberg-products-block
# Add dynamic rendering + SSR to Products by Attribute

See #586 – This PR updates the Products by Attribute block.

### How to test the changes in this Pull Request:

1. Add a Products by Attribute block to a post before activating this PR
2. Build the PR
3. View your existing post, the block should still render correctly
4. Add a new Products by Attribute block to a post, it should work correctly – show the right products, allow for setting attributes in the sidebar, toggling the content, etc.
5. View your post on the frontend, it should display same as the editor preview
